### ISSUE TITLE
Conservatives.com licence

### ISSUE THREAD

**Issue by HelenaByrne** [ukwa/ukwa-ui#359]
We have permission for conservatives.com to be open access. The licence didn't transfer with WCT migration. I tried to edit it manually but it didn't work.

ACT link: https://www.webarchive.org.uk/act/targets/4051#licensing 

Website link: https://www.webarchive.org.uk/wayback/archive/20150317002931/https://www.conservatives.com/ 

@anjackson @nicolabingham @crarugal

**Comment by nicolabingham** [comment]
Hi Both, 
The licence information should be contained in this old Access database bldata (\\ad) T:\Web Archiving Programme\Document Store (Archive Only - Current Docs should be on Wiki)\05 Communications\Enquiries, Permissions & Licensing\Permissions\Web Archiving Permissions.mdb which I can't open right now. Could it wait until @crarugal is back next week? This will tell us when we received permission. 
As far as the issue goes, the licence type shouldn't impact the functionality of manually updating the target to granted?

**Comment by anjackson** [comment]
Unfortunately, a while ago, we decided that we would not allow the 2014-onward licence to be applied without using the permissions workflow. However, it seems like at least the system administrator account should be able to set any licence as needed?

Code section, for reference:

https://github.com/ukwa/w3act/blob/6ad35fff95854edd847c279904534938ff73295d/app/controllers/TargetController.java#L1340-L1343

**Comment by nicolabingham** [comment]
Ah! Thanks Andy that makes sense. I think we should keep the ability to set any licence to Sys Admin please.

**Comment by crarugal** [comment]
![image](https://user-images.githubusercontent.com/18530934/176893512-8936b5d3-d24e-4279-b0eb-006a6eb6f6d2.png)

**Comment by crarugal** [comment]
Licence was granted on 05/11/2004, OK'd for publicity use. The site is still openly accessible with the current selective licence, does it still need to be changed over to the joint LDL licence?
![image](https://user-images.githubusercontent.com/18530934/176894815-1d6913f2-de19-42e8-80ef-76052dd7d38f.png)

**Comment by anjackson** [comment]
Okay, that sounds like it's fine now.  I'll add a separate ticket about archivists being able to add any licence.
